tools/SetupFlow/DevHome.SetupFlow/Views/PackageView.xaml

```diff
@@ -58,22 +59,7 @@
                     </VisualStateManager.VisualStateGroups>
 
                     <ToolTipService.ToolTip>
-                        <StackPanel>
-                            <StackPanel.Resources>
-                                <Style TargetType="TextBlock">
-                                    <Setter Property="Foreground" Value="{ThemeResource TextFillColorSecondary}" />
-                                </Style>
-                            </StackPanel.Resources>
-                            <TextBlock
-                                Foreground="{ThemeResource TextFillColorPrimary}"
-                                FontWeight="Bold"
-                                Text="{Binding TooltipIsInstalled}"
-                                Visibility="{Binding TooltipIsInstalled, Converter={StaticResource StringVisibilityConverter}}"/>
-                            <TextBlock Text="{Binding TooltipName}"/>
-                            <TextBlock Text="{Binding TooltipVersion}"/>
-                            <TextBlock Text="{Binding TooltipSource}"/>
-                            <TextBlock Text="{Binding TooltipPublisher}"/>
-                        </StackPanel>
+                        <controls:PackageDetailsTooltip Package="{Binding}" />

```

Comment:
Whats the nuance here of using 'Binding' instead of 'x:Bind' ?

Reply:
Good question! Since we're inside a `ControlTemplate`, x:Bind will bind to the `TargetType` which must be or inherit from a Control, a Page, or a PageFunctionBase. Hence, to bind to the DataContext instead we use `Binding`.

Reference: [TargetType](https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.controltemplate.targettype?view=windowsdesktop-7.0)